### PR TITLE
Update manage-partners.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line to contact Support if the remove roles option does not show up in the partner center.

*If you do not see the remove roles option, Contact [Partner support](https://partner.microsoft.com/support)*

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6870

File: microsoft-365/commerce/manage-partners.md [hunk 1]
```diff
@@ -90,3 +90,5 @@ You can remove admin roles from a partner at any time. Removing the admin roles
 3. Select the row that contains the name of the partner.
 4. On the partner page, select **Remove roles**.
 5. In the **Remove roles?** dialog box, select **Yes**.
+
+If you do not see the remove roles option, Contact [Partner support](https://partner.microsoft.com/support)
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/manage-partners.md]:
```suggestion
If you do not see the **Remove roles** option, contact the [Partner Center](https://partner.microsoft.com/support).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**